### ISSUE TITLE
feature request: make "send all fund" (using "!") possible when sending to multiple addresses w/o change address

### ISSUE THREAD

**Issue by molecular** [spesmilo/electrum#1863]
For privacy reasons I often make transaction with 1 input and 1 ouptut, sending the full amount (less fee). Electrum has a nice way to allow me to to that:
 * select "send from" on source address, this opens "send" tab with input filled correctly
 * enter an exclamation mark ("!") into amount field
 * observe electrum correctly filling maximum possible amount less fee

Now I ran into a situation where I wanted to send an exact amount and the change somewhere else. I selected an address that had slightly more in it. Now I didn't want the change in my wallet, but instead send the change to someone else (say: an exchange).

Now, I know of the hidden "send to multiple addresses" feature (please don't take it away): one can enter multiple lines of comma-separated <address, amount> pairs into the "pay to" field.

But how to send all remaining fund to one of those addresses? Using a "!" didn't work.

See screenshot to see more easily what I mean.

Is there a good way to do this without making the calculation myself (which is hard because of the unknown fee)?

![image](https://cloud.githubusercontent.com/assets/589745/16714304/f9415078-46be-11e6-9a60-dbe85c3ffeb7.png)

In other words: how to enter the last output amount (or aggregate output amount) marked in blue so that the sum of outputs is the sum of inputs (marked orange) less fee?

My suggestion would be to allow an exclamation mark "!" as amount in the "pay to" field, like this:

```
1AddressA, 2.32041
1AddressB, !
```

**Comment by ecdsa** [comment]
duplicate of #1698
